website/docs/quickstart/flink.md

```diff
@@ -26,6 +26,9 @@ cd fluss-quickstart-flink
 ```
 
 2. Create `docker-compose.yml` file with the following content:
+
+> Please note that if your docker-compose version is lower, you need to add **'version: 2.1'** at the beginning of docker-compose.yaml`
```

Comment:
can we make this more generic? because some users might need versions other than 2.1.. maybe make it:

> **Note:** Depending on the `docker-compose` version installed on your system, you might need to specify a `version` field at the beginning of your `yaml` file.

Reply:
done.